docs/source/adb.rst

```diff
@@ -0,0 +1,131 @@
+.. _adb:
+
+####################
+Android Debug Bridge
+####################
+
+.. important:: Android Debug Bridge is required ONLY for those users pairing an Android mobile device with Cozmo. iOS users require *usbmuxd*; instructions for installing this are given on the install pages for each operating system.
+
+Android Debug Bridge (adb) is necessary in order for the Cozmo SDK to work with Android mobile devices. Please ensure adb is installed before attempting to run the Cozmo SDK with a paired Android device.
+
+-----------------
```

Comment:
Sphinx is issuing a warning about the overline being too short for the title here